### PR TITLE
cpw warns if t < 3 * skin_depth

### PR DESCRIPTION
**Pull request by mhuser**
The losses for microstrip artifacts are over-optimistic when the skin_depth become larger that a third of the strip thickness, at low frequency the skin_depth can be even larger than the thickness. Warns the user.
#648

### CODE REVIEW

File: skrf/media/cpw.py [hunk 1]
```diff
@@ -146,20 +147,39 @@ def alpha_conductor(self) -> NumberLike:
...
+            a_c = ((r_s * sqrt(ep_re)/(480*pi*K(k1)*K_p(k1)*(1-k1**2) ))*\
+                    (1./a * (pi+log((8*pi*a*(1-k1))/(t*(1+k1)))) +\
+                     1./b * (pi+log((8*pi*b*(1-k1))/(t*(1+k1))))))
+        else:
+            a_c = zeros(f.shape)
+            
         a = self.w/2.
         b = self.s+self.w/2.
         K = ellipk      # complete elliptical integral of first kind
         K_p = lambda x: ellipk(sqrt(1-x**2)) # ellipk's compliment
```
Comment (FranzForstmayr) [inline skrf/media/cpw.py:180]: These four lines are not necessary any more

Comment (mhuser) [reply skrf/media/cpw.py]: Indeed, let's get rid of them !

File: skrf/media/cpw.py [hunk 2]
```diff
@@ -146,20 +147,39 @@ def alpha_conductor(self) -> NumberLike:
...
+                
+            a = self.w/2.
+            b = self.s+self.w/2.
+            K = ellipk      # complete elliptical integral of first kind
+            K_p = lambda x: ellipk(sqrt(1-x**2)) # ellipk's compliment
+            a_c = ((r_s * sqrt(ep_re)/(480*pi*K(k1)*K_p(k1)*(1-k1**2) ))*\
+                    (1./a * (pi+log((8*pi*a*(1-k1))/(t*(1+k1)))) +\
+                     1./b * (pi+log((8*pi*b*(1-k1))/(t*(1+k1))))))
+        else:
+            a_c = zeros(f.shape)
```
[the remark this answers is not in the document]
Comment (mhuser) [reply skrf/media/cpw.py:175]: Yes, most simulators are capable of creating a coplanar line with negligible or zero thickness. The formulas have been developed for this simplified case.
When a thickness is specified, the line width and spacing are corrected with an amount and the same formulas with a first-order correction term for zero-thickness are used.
Of course, with zero thickness the conductor loss should be zeros (perfect conductor).
Your question lead me to following observation: the impedance of the scikit-rf cpw is not at all corrected for strip thickness ! When t is specified only conductor losses are affected.

[the remark this answers is not in the document]
Comment (mhuser) [reply skrf/media/cpw.py:175]: open #667 for the specific of the strip thickness correction for impedance, which I believe is out of the scope of this PR

**Comment by mhuser** [comment]
I think we could merge this to close #648 and It is more clean if I address #667 in a separated PR

**Review by jhillairet — APPROVED** [review]

**Comment by mhuser** [comment]
Thanks !

**Pull request merged**